voorhoede/voorhoede-website
# Feat/switch over lustrum and work at pages

## What changes were made

- Remove everything related to the dedicated work-at/lustrum page and models in the codebase and in dato
- Prepare landing pages content to be updated and filled in completely in Dutch as well
- Allow font size to be reduced for section image & text (for the application process block on the work at page)

## How to test or check results

Check:

- https://deploy-preview-688--voorhoede-website.netlify.app/nl/lustrum/
- https://deploy-preview-688--voorhoede-website.netlify.app/en/lustrum/
- https://deploy-preview-688--voorhoede-website.netlify.app/en/work-at
- https://deploy-preview-688--voorhoede-website.netlify.app/nl/work-at

## Checks

## TODO before merge:

- [ ] Update slugs on landing pages on migrated dato environment
- [ ] Publish landing pages
